File: code-coaches/easy/balconies/Balconies.cs
using System;

public class Balconies
{
    static int CalculateArea(string balcony)
    {
        string[] dimensions = balcony.Split(',');
        int length = int.Parse(dimensions[0]);
        int width = int.Parse(dimensions[1]);
        return length * width;
    }

    static void Main(string[] args)
    {
        string balconyA = Console.ReadLine();
        string balconyB = Console.ReadLine();

        int areaA = CalculateArea(balconyA);
        int areaB = CalculateArea(balconyB);

        if (areaA > areaB)
            Console.WriteLine("Apartment A");
        else
            Console.WriteLine("Apartment B");
    }
}

File: code-coaches/easy/balconies/Balconies.swift
func calculateArea(balcony: String) -> Int {
    let dimensions = balcony.split(separator: ",")
    let length = Int(dimensions[0])!
    let width = Int(dimensions[1])!
    return length * width
}

func main() {
    let balconyA = readLine()!
	let balconyB = readLine()!
	
	let areaA = calculateArea(balcony: balconyA)
	let areaB = calculateArea(balcony: balconyB)

	if areaA > areaB {
		print("Apartment A")
	} else {
		print("Apartment B")
	}
}

main()
